rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -144,13 +146,160 @@ void testGetOldFederation(
 
     @Test
     void getOldFederation_previouslySetToNull_returnsNull() {
-        StorageAccessor storageAccessor = new InMemoryStorage();
-        FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+        FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(null);
         federationStorageProvider.setOldFederation(null);
         Federation oldFederation = federationStorageProvider.getOldFederation(federationConstants, activations);
         assertNull(oldFederation);
     }
 
+    private static Stream<Arguments> providePendingFederationAndFormatArguments() {
+        return Stream.of(
+            Arguments.of(P2SH_ERP_FEDERATION_FORMAT_VERSION, new PendingFederationBuilder().build()),
+            Arguments.of(NON_STANDARD_ERP_FEDERATION_FORMAT_VERSION, new PendingFederationBuilder().build()),
+            Arguments.of(STANDARD_MULTISIG_FEDERATION_FORMAT_VERSION, new PendingFederationBuilder().build())
+        );
```

Comment:
```
pendingFed = new PendingFederationBuilder().build();
return Stream.of(
            Arguments.of(P2SH_ERP_FEDERATION_FORMAT_VERSION, pendingFed),
            Arguments.of(NON_STANDARD_ERP_FEDERATION_FORMAT_VERSION, pendingFed),
            Arguments.of(STANDARD_MULTISIG_FEDERATION_FORMAT_VERSION, pendingFed)
        )
```

Reply:
I thought about that, but I wanted a different fed object for each tests. Probably no big deal though.